app/helpers/works_helper.rb

```diff
@@ -191,4 +191,13 @@ def chapter_total_display_with_link(work)
       chapter_total_display(work)
     end
   end
+
+  def get_open_assignments(user)
+    if user == User.orphan_account
+      # orphan_account has too many pseuds to feasibly get. This lets admins access the work form without timing out.
+      return []
+    end
+
+    ChallengeAssignment.by_offering_user(user).undefaulted.unstarted.sent
```

Comment:
I think it'd be cleaner to try to improve the performance of the query more generally, rather than special casing `orphan_account`. If anyone ever feels like adding enough pseuds to rival `orphan_account`, there's nothing stopping them, and they'll still expect their account to work properly.

Reply:
Okay, I can give it a try when I have the time. 

(As an aside, maybe it is good to consider a maximum number of pseuds a non-orphan account user  can create?)